app/src/main/java/app/revanced/integrations/patches/GeneralAdsPatch.java

```diff
@@ -102,6 +103,8 @@ public GeneralAdsPatch() {
     public boolean filter(final String path, final String identifier) {
         BlockResult result;
 
+        shortsCommentBox = path.contains("comment_composer");
```

Comment:
This has nothing to do in the `GeneralAdsPatch` class. Create a seperate one.

Reply:
There's a clean way to read litho component name outside general ads patch, without copy/paste general ads patch?

Reply:
Yes you create a new class and implement the same interface as `GeneralAdsPatch` does, make sure to add it to the same array `GeneralAdsPatch` is added to.

Reply:
If I create a new class, I have no way to independently call it without copy-pasting the ads patch. So:

Can I extend it in the following way?

```public class Something extends GeneralAdsPatch```

Reply:
> Instead you should do: `public final class Something extends Filter`

Oh...ok, I thought about this eventuality, but I wasn't sure if ```filter``` could work on other classes, without being directly called by general ads patch.

Btw...I'll push the new commit tomorrow.